@xen-orchestra/web/src/components/host/HostPifsTable.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <div class="host-pifs-table">
+    <UiTitle type="h4" class="header">
```

Comment:
As said on another PR, there is no `type` prop on this component, and if the class is not used, it can be removed.